src/css/docs-prose.css

```diff
@@ -79,12 +75,6 @@
   .dr-ui--page-layout-aside {
     order: 2;
   }
-
-  /* lessen scale of Aside's OnThisPage links on larger screens */
-  .dr-ui--page-layout-aside .dr-ui--on-this-page li {
-    font-size: 14px;
-    line-height: 1.4;
-  }
```

Comment:
💡 I think we should remove this style, where it makes the font-size on OnThisPage 14px on larger screens. Since the default font-size is a bit larger in Assembly v1, it feels out of place.